Context/DeserializationContext.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Context;
+
+/**
+ * {@inheritdoc}
+ *
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+class DeserializationContext extends AbstractContext implements DeserializationContextInterface
+{
+    /**
+     * Constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
```

Comment:
I think that we can remove this class and create a single context that supports deserialization and serialization. 
I split this class to support the JMSSerializer but we can do it otherwise.